eureka-core/src/main/java/com/netflix/eureka/util/batcher/AcceptorExecutor.java

```diff
@@ -106,35 +104,38 @@ class AcceptorExecutor<ID, T> {
         this.acceptorThread.setDaemon(true);
         this.acceptorThread.start();
 
-        final double[] percentiles = {50.0, 95.0, 99.0, 99.5};
-        final StatsConfig statsConfig = new StatsConfig.Builder()
-                .withSampleSize(1000)
-                .withPercentiles(percentiles)
-                .withPublishStdDev(true)
-                .build();
-        final MonitorConfig config = MonitorConfig.builder(METRIC_REPLICATION_PREFIX + "batchSize").build();
-        this.batchSizeMetric = new StatsTimer(config, statsConfig);
```

Comment:
I wasn't sure what to do with StatsTimer and the docs just say it's "not supported".  Seems like PercentileTimer is a decent replacement

https://netflix.github.io/atlas-docs/spectator/lang/java/servo-migration/#statstimer

Reply:
If percentiles are really needed, then `PercentileTimer` or `PercentileDistributionSummary` would need to be used. Either way, it wouldn't be fully backwards compatible, so the user may need to tune their queries.

I'm not sure sure how this metric is used. The name is `batchSize` which doesn't seem like it would be measuring time, so `PercentileDistributionSummary` may be more appropriate. It was using a timer though.

Reply:
Changed this one to a PercentileDistributionSummary but left another one as percentiletimer but noted that something is going to break if people are using this metric